internal/resources/resources.go

```diff
@@ -78,7 +79,7 @@ func (i *IstioResourcesFinder) FindUserCreatedIstioResources() ([]Resource, erro
 		u.SetGroupVersionKind(resource.GroupVersionKind)
 		err := i.client.List(i.ctx, &u)
 		if err != nil {
-			return nil, err
+			continue
```

Comment:
Maybe some kind of assertion (e.g. k8serrors.IsNotFound) might be possible instead?
Or at least logging the error